py/formak/python.py

```diff
@@ -109,13 +138,21 @@ def __init__(
 
         self.state_size = len(state_model.state)
         self.control_size = len(state_model.control)
+        self.params = {
+            "process_noise": process_noise,
+            "sensor_models": sensor_models,
+            "sensor_noises": sensor_noises,
+            "compile": compile,
+        }
```

Comment:
Don't need to address this now, but params feels like it's something that should be typed (e.g. by a [data class](https://docs.python.org/3/library/dataclasses.html)?)